.idea/codeStyles/Project.xml

```diff
@@ -12,8 +12,5 @@
       <option name="OPTIMIZE_IMPORTS_SORT_NAMES_IN_FROM_IMPORTS" value="true" />
       <option name="OPTIMIZE_IMPORTS_JOIN_FROM_IMPORTS_WITH_SAME_SOURCE" value="true" />
     </Python>
-    <codeStyleSettings language="Python">
```

Comment:
why did that go?

Reply:
This was after updating pycharm. Maybe the new version does not need this?